src/Altinn.Apps/AppTemplates/AspNet/Altinn.App.PlatformServices/Implementation/AppBase.cs

```diff
@@ -430,5 +466,121 @@ private List<string> GetOptionIdsFromFormLayout(string formLayout)
+                    }
+                }
+            }
+        }
+
+        private async Task<StandardBusinessDocument> ConstructStandardBusinessDocument(string instanceGuid)
+        {
+            DateTime completedTime = DateTime.Now;
+
+            Sender digdirSender = new Sender
+            {
+                Identifier = new Identifier
+                {
+                    // 0192 prefix for all Norwegian organisations.
+                    Value = $"0192:{_appSettings.EFormidlingSender}",
+                    Authority = "iso6523-actorid-upis"
+                }
+            };
+
+            List<Receiver> receivers = await GetEFormidlingReceivers();
+
+            Scope scope =
+            new Scope
+            {
+                Identifier = _appMetadata.EFormidling.Process,
+                InstanceIdentifier = Guid.NewGuid().ToString(),
+                Type = "ConversationId",
+                ScopeInformation = new List<ScopeInformation>
+                    {
+                        new ScopeInformation
+                        {
+                            ExpectedResponseDateTime = completedTime.AddHours(2)
+                        }
+                    },
+            };
+
+            BusinessScope businessScope = new BusinessScope
+            {
+                Scope = new List<Scope> { scope }
+            };
+
+            DocumentIdentification documentIdentification = new DocumentIdentification
+            {
+                InstanceIdentifier = instanceGuid,
+                Standard = _appMetadata.EFormidling.Standard,
+                TypeVersion = _appMetadata.EFormidling.TypeVersion,
+                CreationDateAndTime = completedTime,
+                Type = _appMetadata.EFormidling.Type
+            };
+
+            StandardBusinessDocumentHeader sbdHeader = new StandardBusinessDocumentHeader
+            {
+                HeaderVersion = "1.0",
+                BusinessScope = businessScope,
+                DocumentIdentification = documentIdentification,
+                Receiver = receivers,
+                Sender = new List<Sender> { digdirSender }
+            };
+
+            // Skal sikkerhetsnivå kunne konfigureres i appen? 
+            StandardBusinessDocument sbd = new StandardBusinessDocument
+            {
+                StandardBusinessDocumentHeader = sbdHeader,
+                Arkivmelding = new Arkivmelding { Sikkerhetsnivaa = 3 },
+            };
+
+            return sbd;
+        }
+
+        private async void SendEFormidlingShipment(Instance instance, string taskId)
+        {
+            string instanceGuid = instance.Id.Split("/")[1];
+
+            StandardBusinessDocument sbd = await ConstructStandardBusinessDocument(instanceGuid);
+            StandardBusinessDocument sbdVerified = await _eFormidlingClient.CreateMessage(sbd);
```

Comment:
What do we think of error handling here? 
Should we let the app crash or log error and let it continue silently? I guess we have many other options as well.

Reply:
The error is swallowed by ProcessController and the result frontend is "Ukjent feil"